src/Bicep.LangServer.IntegrationTests/TelemetryTests.cs

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Threading.Tasks;
+using Bicep.Core.Syntax;
+using Bicep.Core.Text;
+using Bicep.Core.UnitTests.Assertions;
+using Bicep.LanguageServer.Telemetry;
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json.Linq;
+using OmniSharp.Extensions.LanguageServer.Protocol.Document;
+using OmniSharp.Extensions.LanguageServer.Protocol.Models;
+
+namespace Bicep.LangServer.IntegrationTests
+{
+    [TestClass]
+    [SuppressMessage("Style", "VSTHRD200:Use \"Async\" suffix for async methods", Justification = "Test methods do not need to follow this convention.")]
+    public class TelemetryTests
+    {
+        [NotNull]
+        public TestContext? TestContext { get; set; }
+
+        [TestMethod]
+        public async Task ValidateDeclarationSnippetCompletionItemContainsCommandWithTelemetryInformation()
```

Comment:
In this test, you're asserting that a completion command contains the necessary data to trigger a telemetry event, but you're not testing the next part (client sends `workspace/executeCommand`, server sends a `telemetry/event` with the correct data to the client).

Could you either expand this test to add a full end-to-end, or add a new test to assert this?

Reply:
Updated the test here:
https://github.com/Azure/bicep/commit/e4dd5cb53c39092925cf801a49c8ec58751f3666